python/dataset.cpp

```diff
@@ -259,25 +259,33 @@ void init_dataset(py::module &m) {
   m.def("concatenate",
         py::overload_cast<const DataConstProxy &, const DataConstProxy &,
                           const Dim>(&concatenate),
+        py::arg("x"), py::arg("y"), py::arg("dim"),
         py::call_guard<py::gil_scoped_release>(), R"(
         Concatenate input data array along the given dimension.
 
         Concatenates the data, coords, labels and masks of the data array.
         Coords, labels and masks for any but the given dimension are required to match and are copied to the output without changes.
 
+        :param x: First DataProxy.
+        :param y: Second DataProxy.
```

Comment:
DataArray? Technically it isn't, but the term `DataProxy` is not used in the documentation, because the distinction is usually not relevant for a Python user.